smart_open/webhdfs.py

```diff
@@ -112,27 +133,29 @@ def readline(self):
 
 
 class BufferedOutputBase(io.BufferedIOBase):
-    def __init__(self, uri_path, min_part_size=WEBHDFS_MIN_PART_SIZE):
+    def __init__(self, uri, min_part_size=WEBHDFS_MIN_PART_SIZE):
```

Comment:
What has changed here? How is `uri` different from `uri_path`?

Also, please document the parameters in the docstring.

Reply:
For consistency, as this parameter is called uri in BufferedInputBase. Also - this is whole uri, not only path. What about changing its name to 'http_uri'?